### DISCUSSION TITLE
[knitr] Objects: Objects to manipulate options, patterns and hooks | /knitr/objects/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#786] [Comments]
https://yihui.org/knitr/objects/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
That setting sounds unusual... Anyway, use opts_chunk$set(comment = '>') if you really like the output has a prefix >. I suppose you probably want opts_chunk$set(prompt = TRUE), though.

> Originally posted on 2012-11-25 19:48:15

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
Sorry, I guess I cannot help without a minimal reproducible example... in theory, child documents are included in parent documents, so maxwidth must be applied to them.

> Originally posted on 2013-08-22 04:22:35

[a comment by an automated account is not shown]

**Comment by zhipengliux** [comment]
How to add a "copy to clipboard" buttom in R markdown
